### ISSUE TITLE
[Bug] 3.4.1 Route Map multiple entities bug

### ISSUE THREAD

**Issue by Borch97** [thingsboard/thingsboard#7380]
**Describe the bug**
We have made a platform update to version 3.4.1 in order to use latest values in time series widgets.

When updating successfully, we have tried to perform a test using the "route map" widget. When entering a single entity, it does so correctly, but when using an alias that groups more than one entity, it stops receiving data from both telemetry and last values.

One entity:
![Screenshot from 2022-10-05 12-17-22](https://user-images.githubusercontent.com/16107693/194038005-d1ac06e8-1bc0-475b-a371-ae21beb53808.png)

Multiple entities:
![Screenshot from 2022-10-05 12-17-31](https://user-images.githubusercontent.com/16107693/194038051-641b38f7-e985-4b61-9d27-73eb3b0f07e1.png)

Is it a bug or is it an intended functionality?

**Your Server Environment**

Thingsboard 3.4.1 CE

**Comment by KenSense** [comment]
Check if all your devices have the correct attribute information to be shown on the map.
Easy way to check is to put a table on your dashboard with this info.

**Comment by Borch97** [comment]
Yes, i checked and all devices have selected attribute. To re-check, i change the latest value attribute to "name" attribute, auto-generated by device and the map keeps stuck in "loading..."
![Screenshot from 2022-10-06 19-24-45](https://user-images.githubusercontent.com/16107693/194379163-db15601d-3a8e-49bb-ad4a-b9966b827459.png)


Checking the network tab on chrome development debug window, i get all 200 and websocket stuck in pending with the following messages.
![Screenshot from 2022-10-06 19-25-13](https://user-images.githubusercontent.com/16107693/194379150-cf451264-1fe0-4eeb-bca1-65645b218692.png)

**Comment by DRazumovskiy** [comment]
I have the same bug with ThingsBoard Cloud version. When mixing timeseries and latest data in one visualization of route map the widget does not load data, but shows "Loading" message
